public/docs/ts/latest/guide/forms.jade

```diff
@@ -348,41 +351,28 @@ figure.image-display
 
 .l-main-section
 :marked
-  ## Track change-state and validity with **ngControl**
+  ## Track change-state and validity with **ngModel**
 
   A form isn't just about data binding. We'd also like to know the state of the controls on our form.
 
-  By setting `ngControl` we create a directive that can tell if the user touched the control,
-  if the value changed, or if the value became invalid.
+  Using `ngModel` in a form gives us more than just two way data binding. It also tells us if the user touched the control, if the value changed, or if the value became invalid.
 
-  This directive doesn't just track state; it updates the control with special
-  Angular CSS classes from the set we listed above.
+  `ngModel` doesn't just track state; it updates the control with special Angular CSS classes from the set we listed above.
   We can leverage those class names to change the appearance of the
   control and make messages appear or disappear.
 
   We'll explore those effects soon. Right now
-  we should **add `ngControl` to all three form controls**,
+  let's make sure we have `ngModel` and the corresponding name attribute on all three form controls,
   starting with the *Name* input box
 +makeExample('forms/ts/app/hero-form.component.html', 'ngControl-1', 'app/hero-form.component.html (excerpt)')(format=".")
 :marked
-  We set this particular `ngControl` to "name" which makes sense for our app. Any unique value will do.
+  We set the `name` attribute to "name" which makes sense for our app. Any unique value will do.
 
 .l-sub-section
   :marked
-    Internally Angular creates `Controls` and registers them under their `ngControl` names
-    with an `NgForm` directive that Angular attached to the `<form>` tag.
+    Internally Angular creates `FormControls` and registers them with an `NgForm` directive that Angular attached to the `<form>` tag. Each `FormControl` is registered under the name we assigned to the `name` attribute.
     We'll talk about `NgForm` [later in the chapter](#ngForm).
 
-    The `ngControl` *attribute* in our template actually maps to the 
-    [NgControlName](../api/common/NgControlName-directive.html) directive.
-    There is also a `NgControl` *abstract* directive which is *not the same thing*.
-    We often ignore this technical distinction and refer to `NgControlName` more conveniently (albeit incorrectly) as the *NgControl* directive.
-    
-    While we're under the hood, we might as well note that the `ngModel` in the 
-    two-way binding syntax is now a property of the `NgControlName` directive. 
-    The `NgModel` directive is no longer involved. We only need one directive to manage the DOM element
-    and there is no practical difference in the way either directive handles data binding.     
-
```

Comment:
It's a bit unclear if we should replace this with alternative details. It's my understanding that the features of ngControl has been added to ngModel, so I assume this section is no long relevant, but we might want to confirm.